packages/ember-htmlbars/tests/integration/streams_test.js

```diff
@@ -0,0 +1,56 @@
+import Helper from "ember-htmlbars/system/helper";
+import { registerHelper } from "ember-htmlbars/helpers";
+import run from 'ember-metal/run_loop';
+import EmberView from 'ember-views/views/view';
+import EmberObject from 'ember-runtime/system/object';
+import compile from 'ember-template-compiler/system/compile';
+import { runAppend, runDestroy } from "ember-runtime/tests/utils";
+import { set } from 'ember-metal/property_set';
+import { isStream } from 'ember-metal/streams/utils';
+
+var view;
+
+QUnit.module('ember-htmlbars: streams', {
+  setup() {
+
+  },
+
+  teardown() {
+    runDestroy(view);
+    view = null;
+  }
+});
+
+QUnit.test("Unbound helpers should be unbound when accessed directly and when accessed in a subexpression", function() {
```

Comment:
This test name is confusing to me.  It talks about using unbound helpers, but the test is confirming that the output updates when the value it is retrieving changes (which to me seems like a bound helper).